--- warsow-0.5/source/Makefile~	2009-03-29 17:18:14.000000000 +0200
+++ warsow-0.5/source/Makefile	2009-08-31 16:51:13.000000000 +0200
@@ -201,6 +201,7 @@
 else
 CFLAGS_COMMON+=$(CFLAGS_RELEASE)
 endif
+CFLAGS_COMMON+=$(CFLAGS)
 CFLAGS_OPENAL=-I/System/Library/Frameworks/OpenAL.framework/Headers
 CFLAGS_QF=-I/Library/Frameworks/SDL.framework/Headers
 CFLAGS_SDL=-I/Library/Frameworks/SDL.framework/Headers
@@ -213,6 +214,7 @@
 LDFLAGS_QF=-framework SDL -framework Ogg -framework Vorbis 
 LDFLAGS_OPENAL=-framework OpenAL -framework Ogg -framework Vorbis
 endif
+LDFLAGS_COMMON+=$(LDFLAGS)
 
 ######################################################################################################
 # Helper functions
