--- warsow-0.5/source/Makefile~	2009-09-01 16:46:03.000000000 +0200
+++ warsow-0.5/source/Makefile	2009-09-01 16:45:41.000000000 +0200
@@ -135,7 +135,7 @@
 X11BASE?=/usr/X11R6
 
 CFLAGS_COMMON=$(CFLAGS) -pipe -I. -I$(LOCALBASE)/include -I$(X11BASE)/include -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Wall
-CFLAGS_RELEASE=-O2 -fno-strict-aliasing -ffast-math -funroll-loops -DNDEBUG
+CFLAGS_RELEASE=-ffast-math -funroll-loops -DNDEBUG
 CFLAGS_DEBUG=-O0 -ggdb -D_DEBUG
 ifeq ($(DEBUG_BUILD),YES)
 CFLAGS_COMMON+=$(CFLAGS_DEBUG)
@@ -152,7 +152,7 @@
 ifeq ($(OS),FreeBSD)
 LDFLAGS_COMMON=-L/usr/local/lib -lm -pthread
 else
-LDFLAGS_COMMON=-L/usr/local/lib -ldl -lm -Wl -O1 --as-needed
+LDFLAGS_COMMON=-L/usr/local/lib -ldl -lm
 endif
 
 ifeq ($(ARCH),x86_64)
@@ -201,6 +201,7 @@
 else
 CFLAGS_COMMON+=$(CFLAGS_RELEASE)
 endif
+CFLAGS_COMMON+=$(CFLAGS)
 CFLAGS_OPENAL=-I/System/Library/Frameworks/OpenAL.framework/Headers
 CFLAGS_QF=-I/Library/Frameworks/SDL.framework/Headers
 CFLAGS_SDL=-I/Library/Frameworks/SDL.framework/Headers
@@ -213,6 +214,7 @@
 LDFLAGS_QF=-framework SDL -framework Ogg -framework Vorbis 
 LDFLAGS_OPENAL=-framework OpenAL -framework Ogg -framework Vorbis
 endif
+LDFLAGS_COMMON+=$(LDFLAGS)
 
 ######################################################################################################
 # Helper functions
